src/main/webapp/app/entities/lecture/lecture-attachments.component.ts

```diff
@@ -131,19 +131,22 @@ export class LectureAttachmentsComponent implements OnInit {
 
     downloadAttachment(downloadUrl: string) {
```

Comment:
I have not changed this file and can't remove this change by checking it out again from develop - must be our pre-commit hook.